### DISCUSSION TITLE
<what is future with github >

### DISCUSSION THREAD

**Discussion by govindrathod55** [community/community#45473] [New to GitHub]
### Select Topic Area

Question

### Body

Hello Github Family 

Namaste 🙏

I am from India 🇮🇳

I am new on this platform. Is there anyone from india who can guide me properly so i can guide my team and sharing more awareness about this platform but before that i need to understand whole things about it .

If anyone can guide me it will be really helpful for me 

Thank You
Regards
Govind Rathod 🤝

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @govindrathod55  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
